### PR TITLE
Display a message when a visit will be rejected

### PR DESCRIPTION
**Pull request by chrishanes**

### CODE REVIEW

File: app/views/prison/visits/_visitor_details.html.erb [hunk 1]
```diff
@@ -5,14 +5,14 @@
 
 <% if Nomis::Feature.contact_list_enabled?(@visit.prison_name) && !@visit.contact_list_unknown? %>
```
Comment (chrishanes) [inline app/views/prison/visits/_visitor_details.html.erb]: This has been changed in https://github.com/ministryofjustice/prison-visits-2/pull/864 which needs to be merged before release

File: app/views/prison/visits/_issue_with_prisoner_section.html.erb [hunk 2]
```diff
@@ -64,13 +64,13 @@
...
           <%= t('.prisoner_released') %>
         </label>
       <% end %>
       <%= error_container(rf, :reasons) do %>
         <label class="block-label">
-          <%= rf.object.checkbox_for(:prisoner_banned, class: 'js-Conditional') %>
+          <%= rf.object.checkbox_for(:prisoner_banned, class: 'js-Rejection', 'data-rejection-el': 'rejection-message') %>
           <%= t('.prisoner_banned') %>
         </label>
       <% end %>
```
Comment (doctorpod) [inline app/views/prison/visits/_issue_with_prisoner_section.html.erb]: There seems to be a repeating pattern here with only the field names changing. Could we have a loop go through an array of field names or use a partial here?

Comment (chrishanes) [inline app/views/prison/visits/_issue_with_prisoner_section.html.erb]: That sounds like a good idea, but beyond my knowledge.

Comment (chrishanes) [inline app/views/prison/visits/_issue_with_prisoner_section.html.erb]: @doctorpod Can we accept the current partial and make a card in Trello inbox for refactor or should we do this now? If the later can you help please?

Comment (doctorpod) [inline app/views/prison/visits/_issue_with_prisoner_section.html.erb]: Lets do it later - I'll make a card

Comment (kyliehavelock) [inline app/views/prison/visits/_issue_with_prisoner_section.html.erb]: @doctorpod can you point me to the card?

**Pull request merged**